app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -365,6 +368,19 @@ module.exports = angular.module('spinnaker.core.pipeline.config.pipelineConfigur
       return msg;
     };
 
+    this.getPipelineExecutions = () => {
+      executionService.getExecutionsForConfigIds($scope.pipeline.application, $scope.pipeline.id, 5)
```

Comment:
This will fetch the five newest executions. Is that a sensible number, or should I use something else?